adv.py

```diff
@@ -1049,7 +1049,7 @@ def get_obj(x):
 
     def draw(self, win):
         for y, row in enumerate(self.B):
-            for x, cell in enumerate(row):
+            for x, cell in reversed([*enumerate(row)]):
```

Comment:
I think it would be clearer to do reversed(list(enumerate(row))), it seems to do the same thing?

Reply:
Sure. It does the same thing indeed.